gatsby_fp/src/components/fieldComponents/genInfoBio.js

```diff
@@ -0,0 +1,13 @@
+import React from "react"
```

Comment:
Some of these section components have the same general shape: one prop, rendered inside a paragraph, wrapped by a common `className`. Perhaps there's a way to generalize them into a Section component? The ability to reuse components is almost always a good thing.

Reply:
I guess it would have two props: the field label, and the field data. Something like `<Section fieldLabel="Appointments" fieldData={facultyData.appointment} />`?

Reply:
There are other ways, but that's definitely the most straightforward.